### PR TITLE
Added first API endpoints, and mock persistence layer

### PR DESCRIPTION
**Pull request by rob-uniqueword**

### CODE REVIEW

File: ranko.go [hunk 1]
```diff
@@ -2,8 +2,13 @@ package main
 
 import (
 	"fmt"
+
+	"github.com/LettsDoSomeCoding/ranko/api"
 )
 
+var greeting = "Hello I'm Ranko. I don't have much of a brain yet, but maybe I could remember a player or two?"
```
Comment (LettsDoSomeCoding) [inline ranko.go:9]: Noice

File: api/api.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+var homePageGreeting string
+
+func StartServer(greeting string) {
+	fmt.Println("Starting server")
+
+	homePageGreeting = greeting
+	router := mux.NewRouter().StrictSlash(true)
+
+	router.HandleFunc("/", homePage)
+	router.HandleFunc("/player/{id}", createPlayerHandler).Methods("POST")
```
Comment (LettsDoSomeCoding) [inline api/api.go]: So this mux package is used to define all our routing behaviour?

**Review by LettsDoSomeCoding — CHANGES_REQUESTED** [review]
Looks good overall. Could you also add some brief instructions in the readme on how to start the app up locally so that I can take a look at it in action?

File: api/api.go (hunk 2 above)
Comment (rob-uniqueword) [reply api/api.go]: Yep, that's the plan at the moment. It just seems to add a bunch of options for fancier routing and better handling of parameters over the http package's standard router.

**Review by LettsDoSomeCoding — APPROVED** [review]

**Pull request merged**